docs/components/multiselect-field/index.md

```diff
@@ -169,23 +175,21 @@ An example with translations may be something like:
 </Form::Fields::Multiselect>
 ```
 
-## No Results Block
+## No results text
 
-Required.
+`@noResultsText` is shown when there are no results after filtering. 
 
-A `:noResults` block is required and exposed to allow consumers to specify text when there are no results after filtering the options. Since it is a named block, any content can be rendered inside; however, we recommend only putting text as the content.
+Required.
```

Comment:
I could be wrong but I think in other examples we put `Required.` at the top of the description if we want to be consistent here with other docs.

Reply:
Nice.  In this case then, I'd say go with whatever feels best and then we can decide and update in a downstream PR later!